### DISCUSSION TITLE
How to initialize the data from backend to frontend at program launch

### DISCUSSION THREAD

**Discussion by dataflowjs** [wailsapp/wails#2259] [Q&A]
### Have you read the Documentation Contribution Guidelines?

- [X] I have read the [Documentation Contribution Guidelines](https://wails.io/community-guide#ways-of-contributing).

### Description

what i need to do in doReady func or ...?
```go
func (a App) domReady(ctx context.Context) {
	
}
```
and in index.html

```js
<script></script>
```
Cant understand how to send data from ```go``` to ```js``` when dom ready and window is opened  

### Self-service

- [x] I'd be willing to address this documentation request myself.

**Comment by lyimmi** [comment]
Hi,
I think you should read the "[How does it work](https://wails.io/docs/howdoesitwork)?" part of the docs!

You can call a bound method from the frontend, or dispatch an event OnDomReady from go and catch it on the frontend.

The following sections cover what you are asking for.
[Method Binding](https://wails.io/docs/howdoesitwork#method-binding)
[Calling bound Go methods](https://wails.io/docs/howdoesitwork#calling-bound-go-methods)
